cartographer_ros/cartographer_ros/offline_node_main.cc

```diff
@@ -175,7 +175,7 @@ void Run(const std::vector<string>& bag_filenames) {
     rosbag::Bag bag;
     bag.open(bag_filename, rosbag::bagmode::Read);
     rosbag::View view(bag);
-    const ::ros::Time begin_time = view.getBeginTime();
+    const ::builtin_interfaces::msg::Time begin_time = view.getBeginTime();
```

Comment:
Yeah, again, this is a file that isn't compiled, and can't be ported/compiled until we have rosbag support in ROS2.  So we have the same decision to make as for assets_writer_main.cc